cosmos/profiles/clickhouse/user_pass.py

```diff
@@ -0,0 +1,55 @@
+"Maps Airflow Postgres connections using user + password authentication to dbt profiles."
+from __future__ import annotations
+
+from typing import Any
+
+from ..base import BaseProfileMapping
+
+
+class ClickhouseUserPasswordProfileMapping(BaseProfileMapping):
+    """
+    Maps Airflow generic connections using user + password authentication to dbt Clickhouse profiles.
+    https://docs.getdbt.com/docs/core/connect-data-platform/clickhouse-setup
+    """
+
+    airflow_connection_type: str = "generic"
+    default_port = 9000
+
```

Comment:
Please, could you add the attribute:
```
is_community = True
```
https://github.com/astronomer/astronomer-cosmos/blob/3786703609e69c1e8f4b2db1475fe8b6ea00a117/cosmos/profiles/base.py#L33C31-L33C31